package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "dengue-map",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc --noEmit && vite build",
+    "preview": "vite preview",
+    "lint-fix": "eslint . --fix --ignore-path .gitignore",
+    "lint": "eslint . --ignore-path .gitignore"
```

Comment:
I've already included prettier in the linting rules, rather than making it a separate process. I think it should amount to the same thing shouldn't it?